charts/backstage/values.yaml

```diff
@@ -162,9 +162,11 @@ service:
   ##
   type: ClusterIP
   ## @param service.ports.backend Backstage svc port for client connections
+  ## @param service.ports.targetPort Backstage svc target port referencing receiving pod container port
   ##
   ports:
     backend: 7007
+    targetPort: backend
```

Comment:
Could we add a description to this? It's an obvious one I admit, but it feels weird having blank descriptions in the README.md values table.. what do you think?

Reply:
I actually added it a few lines earlier 

```
  ## @param service.ports.targetPort Backstage svc target port referencing receiving pod container port
```

However, it appears there is an issue where there are a mix of ways value parameters are being documented so there are missing descriptions. Recommend integrating this as is and performing a bugfix followup to align the description.

Thoughts?

Reply:
Yep, I commented on this PR before I saw [this one](https://github.com/backstage/charts/pull/30) that goes through and adds a description and corrects the value parameter tags so that the `helm-docs` picks it up nicely. So for the `targetPort` we can probably do `# -- Backstage svc target port referencing receiving pod container port`

Reply:
Updated